usaspending_api/awards/models.py

```diff
@@ -291,6 +305,7 @@ class FinancialAssistanceAward(AwardAction):
     reporting_period_start = models.DateField(blank=True, null=True)
     reporting_period_end = models.DateField(blank=True, null=True)
     last_modified_date = models.DateField(blank=True, null=True)
+    submitted_type = models.CharField(max_length=1, blank=True, null=True, verbose_name="Submitted Type")
```

Comment:
I have no idea what this is, it was a change from @alexjajabah and @playmakervi 's changes